### PR TITLE
Move OpenSSL-Query from omc-tools

### LINKED ISSUE #134: addrev script can't locate OpenSSL/Query/REST.pm
It seems like my addrev script stopped working a while ago. (Don't know when, haven't been using it for a while).

```
msp@office:~/src/openssl/master$ addrev --prnum=20106 @beldmit
Rewrite 8eaa815234dc00804c07d6ab7a1e5d7283c81df8 (1/1) (0 seconds passed, remaining 0 predicted)    Can't locate OpenSSL/Query/REST.pm in @INC (you may need to install the OpenSSL::Query::REST module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.32.1 /usr/local/share/perl/5.32.1 /usr/lib/x86_64-linux-gnu/perl5/5.32 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.32 /usr/share/perl/5.32 /usr/local/lib/site_perl) at /home/msp/openssl/tools/review-tools/gitaddrev line 9.
BEGIN failed--compilation aborted at /home/msp/openssl/tools/review-tools/gitaddrev line 9.
msg filter failed: gitaddrev --prnum=20106 --reviewer=@abc --myemail=xxx@ncp-e.com
 
addrev failed
Died at /home/msp/openssl/tools/review-tools/addrev line 89.
```

I checked that my tools directory is up-to-date
```
msp@office:~/src/openssl/master$ which addrev
/home/msp/openssl/tools/review-tools/addrev
msp@office:~/src/openssl/master$ cd /home/msp/openssl/tools/
msp@office:~/openssl/tools$ git log --oneline -1
ec24476 pick-to-branch: fix exit message by making variable name consistent
```
and the review-tools/README tells me to look for a README.md in the OpenSSL-Query folder:

https://github.com/openssl/tools/blob/ec24476e796cd39ea6b44ae8139b768a4a166c43/review-tools/README#L6

The README.md however has been removed in commit 0d8b3195d26d3382e4580633b49e0626225dfdad by @levitte.

What am I missing?

### PR DESCRIPTION
**Pull request by levitte**
Fixes #134

### CODE REVIEW

**Review by mspncp — APPROVED** [review]
LGTM: both tree hashes of OpenSSL-Query are identical (3c2c7c4de4f20ea525ee40fe409446d66997dd16)

**omc-tools**
```
msp@debian:~/openssl/omc-tools$ git show --oneline --no-patch 
463e1f6 (HEAD -> master, origin/master, origin/HEAD) README.md: minor enhancements
msp@debian:~/openssl/omc-tools$ git rev-parse HEAD:OpenSSL-Query
3c2c7c4de4f20ea525ee40fe409446d66997dd16
```
**tools**
```
msp@debian:~/openssl/tools$ git show --oneline --no-patch 
4127ad4 (HEAD -> gh-170, github/gh-170) Move OpenSSL-Query from omc-tools
msp@debian:~/openssl/tools$ git rev-parse HEAD:OpenSSL-Query
3c2c7c4de4f20ea525ee40fe409446d66997dd16
```

File: OpenSSL-Query/Makefile.PL [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+license     'apache';
+
+perl_version 5.006;
+
+tests_recursive('t');
+
+resources (
+   license    => 'http://www.apache.org/licenses/LICENSE-2.0',
+   repository => 'https://github.com/openssl/omc-tools.git',
+   bugtracker => 'https://github.com/openssl/omc-tools/issues',
```
Comment (mspncp) [inline OpenSSL-Query/Makefile.PL:20]:
```suggestion
   repository => 'https://github.com/openssl/tools.git',
   bugtracker => 'https://github.com/openssl/tools/issues',
```

**Review by mspncp — CHANGES_REQUESTED** [review]
One nit:

File: OpenSSL-Query/Makefile.PL (hunk 1 above)
Comment (mspncp) [reply OpenSSL-Query/Makefile.PL:20]: found by

    git diff 0d8b3195d26d3382e4580633b49e0626225dfdad~:OpenSSL-Query HEAD:OpenSSL-Query

**Review by mspncp — APPROVED** [review]

**Comment by levitte** [comment]
@openssl/otc, another review needed please

**Review by paulidale — APPROVED** [review]

**Comment by levitte** [comment]
Merged

e31cc768643d8ea5b8974f1f871bceaeeca3718c Move OpenSSL-Query from omc-tools

**Pull request closed without merging**